sdk-extensions/autoconfigure/src/testOtlp/java/io/opentelemetry/sdk/autoconfigure/OtlpGrpcRetryTest.java

```diff
@@ -120,6 +120,10 @@ private static <T> void testRetryableStatusCodes(
 
       CompletableResultCode resultCode =
           exporter.apply(dataSupplier.get()).join(10, TimeUnit.SECONDS);
+      assertThat(resultCode.isDone())
+          .as("Exporter didn't complete in time. Consider increasing join timeout?")
+          .isTrue();
```

Comment:
Seems reasonable to me. Can you add the same to `OtlpHttpRetryTest#testRetryableStatusCodes` which has the same bit of logic?

Reply:
I think you meant `testDefaultRetryPolicy` but yeah. Will do.

Reply:
Oh I actually meant the separate test class `OtlpHttpRetryTest`.

Reply:
Ah, yeah, I'll check it out too....thanks.